src/client/components/Swipe/Swipe.jsx

```diff
@@ -171,25 +196,24 @@ class Swipe extends Component {
 
     return (
       <div className="swipe">
+        <div className="swipe-header">
+          <BackButton
+            onClick={() => {
+              this.routeChange(PATHS.trips(userId));
+            }}
+          />
+          <div className="city">{this.state.city}</div>
+          <img
+            className="icon-list"
+            src="/assets/common/icon-list.png"
+            onClick={() => {
+              this.routeChange(PATHS.list(userId, tripId));
```

Comment:
you don't need to enclose with curly brackets if it's just one line. makes it easier to read the element's props.

```js
onClick={() => this.routeChange(PATHS.list(userId, tripId))}
```